src/Routes.js

```diff
@@ -46,7 +44,6 @@ export const AccountStatContext = createContext({
 });
 
 export const Routes = () => {
-  const searchParams = useMemo(() => getSearchParams(), []);
```

Comment:
We shouldn't pass this in here. all components that use params lower in the tree should access theme vie the react router (params) hook